src/answer/answer.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@nestjs/common';
+import { AnswerRepository } from './answer.repository';
+import { ResponseRepository } from '../response/response.repository';
+import { CreateAnswerDto } from './dto/create-answer.dto';
+import { UpdateAnswerDto } from './dto/update-answer.dto';
+import { Response } from '../response/response.entity'
+import { Answer } from './answer.entity';
+
+@Injectable()
+export class AnswerService {
+    constructor(
+        private readonly answerRepository: AnswerRepository,
+        private readonly responseRepository: ResponseRepository
+    ) {}
+
+    async createAnswer(dto: CreateAnswerDto) {
+        let response = await this.responseRepository.findOne({ where: { user_uuid: dto.user_uuid }, relations: ["answers"] });
+        if (!response) {
+            response = await this.responseRepository.createResponse({ user_uuid: dto.user_uuid });
```

Comment:
let -> const

const response = await this.responseRepository.findOrCreateResponse(~~);